pkg/lambdaworks/lambdaworks.go

```diff
@@ -1,12 +1,113 @@
-// Wrapper around the lambdaworks library. A mock, for now.
 package lambdaworks
 
 /*
 #cgo LDFLAGS: pkg/lambdaworks/lib/liblambdaworks.a -ldl
 #include "lib/lambdaworks.h"
+#include <stdlib.h>
 */
 import "C"
+import (
+	"unsafe"
+)
 
-func Number() int {
-	return int(C.number())
+// Go representation of a single limb (unsigned integer with 64 bits).
+type Limb C.limb_t
+
+// Go representation of a 256 bit prime field element (felt).
+type Felt struct {
+	limbs [4]Limb
+}
+
+// Converts a Go Felt to a C felt_t.
+func (f Felt) toC() C.felt_t {
+	var result C.felt_t
+	for i, limb := range f.limbs {
+		result[i] = C.limb_t(limb)
+	}
+	return result
+}
+
+// Converts a C felt_t to a Go Felt.
+func fromC(result C.felt_t) Felt {
+	var limbs [4]Limb
+	for i, limb := range result {
+		limbs[i] = Limb(limb)
+	}
+	return Felt{limbs: limbs}
+}
+
+// Gets a Felt representing the "value" number, in Montgomery format.
+func From(value uint64) Felt {
```

Comment:
```suggestion
func FeltFromUint64(value uint64) Felt {
```
Lets make this name more descriptive